aten/src/ATen/core/Vitals.h

```diff
@@ -35,11 +50,19 @@ struct TORCH_API TorchVital {
   TorchVital() = delete;
 
   TorchVitalAttr& create(const std::string& attr);
+  TorchVitalAttr& create(const std::string& attr, bool force);
 
-  ~TorchVital() {
+  std::string toString() {
```

Comment:
Why toString function instead of redefining `<<` so that you can directly call std::cout << this ?